content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Poseidon-Firmament - An alternate scheduler
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+**Current release of Poseidon-Firmament scheduler is an <code> alpha </code> release.**
+ 
+Poseidon-Firmament scheduler is an alternate scheduler alongside the default Kubernetes scheduler. 
+
+{{% /capture %}}
```

Comment:
I re-read the page. Looking better.
I still think this sentence could use more tweaks:
Poseidon-Firmament scheduler is an alternate scheduler that can  be deployed alongside the default Kubernetes scheduler.
OR
Poseidon-Firmament scheduler is a Kubernetes compatible scheduler that can be used alongside the default Kubernetes scheduler.

Someone else will have to provide a lgtm and approve.
What happens with the doc that was merged to the 1.1.4 branch?

Reply:
Changed the line.  
I am not sure about this.
When we had merged the PR into the dev-1.14 branch the document was very different and has since been updated.
@bsalamat could you please advise on how to proceed. 
Do we have to bring the dev-1.14 branch up-to-date with master ?

/assign @tengqm 
/assign @stewart-yu